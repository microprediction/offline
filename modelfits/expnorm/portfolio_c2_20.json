{"g1": 0.4420702828429293, "g2": 0.06486110332857538, "logK": -4.864997574987787, "loc": 0.14995751538683902, "logScale": 1.004973267542165}