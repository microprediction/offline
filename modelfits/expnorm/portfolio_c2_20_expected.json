{"g1": 0.3351014034820124, "g2": 13.74603482032844, "logK": 0.9573967811250048, "loc": -0.09510600483968117, "logScale": -1.9425766783835234}